docs/UserGuide.md

```diff
@@ -147,6 +147,42 @@ Clears all entries from the address book.
 
 Format: `clear`
 
+### Adding new Project : `addP`
+
+Adds a new project in TaskHub
+
+Format: `addP p/PROJECT_NAME [e/EMPLOYEE_NAME] ...`
+
+* Adds a new project with the employees assigned to the project.
+* The employee must exist in the employees list. 
+
+Examples: 
+* `addP p/Project1 e/Betsy` will add `Project1` to the projects list with `Betsy` assigned to the project.
+* `addP p/Project2` will add an empty `Project2` to the projects list.
+
+### Assign employee(s) to project: `assignE`
+
+Assigns employee(s) to a project in TaskHub
+
+Format: `assignE p/PROJECT_NAME e/EMPLOYEE_NAME [e/MORE_EMPLOYEE_NAMES]...`
+* The employee(s) will be assigned to the project
+* The project name and employee names must exist in TaskHub.
+
+Examples:
+* `assignE p/Project1 e/Anton e/Joe` will add `Anton` and `Joe` to `Project1`
+
+### Delete Project: `deleteP`
```

Comment:
`Delete Project` should be `Delete a project`.  Note the capitalization of `project`.